flepimop/gempyor_pkg/src/gempyor/simulate.py

```diff
@@ -359,8 +243,8 @@ def simulate(
             f">>> {seir_modifiers_scenario}_{outcome_modifiers_scenario} completed in {time.monotonic() - start:.1f} seconds"
         )
 
-
 if __name__ == "__main__":
+    warnings.warn("This function is deprecated, use the CLI instead: `flepimop simulate ...`", DeprecationWarning)
     simulate()
```

Comment:
This doesn't appear when calling `gempyor-simulate` directly. What about doing something like:
```python
@cli.command(params=[config_files_argument] + list(config_file_options.values()), deprecated=True)
@click_helpstring([config_files_argument] + list(config_file_options.values()))
def simulate_deprecated(**kwargs: Any) -> None:
    simulate(**kwargs)
simulate_deprecated.__doc__ = simulate.__doc__
```
and then pointing the `gempyor-seir` and `gempyor-simulate` entry points to this thin wrapper?

Reply:
hadn't fixed this yet, but makes sense to me. will add.